### PR TITLE
refactor: add introduction for templates in readme's

### PR DESCRIPTION
**Pull request by TeamWolfyta**

### CODE REVIEW

File: 3-blank/README.md [hunk 1]
```diff
@@ -88,7 +92,7 @@ For the full command list please view the [Docker Documentation](https://docs.do
 
 - [Check frequently asked questions](https://discord-ts.js.org/docs/faq)
 - [Check examples](https://github.com/oceanroleplay/discord.ts/tree/main/packages/discordx/examples)
-- Ask in [Discord Server](https://discord-ts.js.org/discord)
+- Join and Ask in our [Discord Server](https://discord-ts.js.org/discord)
```
Comment (oceanroleplay) [inline 3-blank/README.md:95]: revert this for all, I found the original one better

Comment (TeamWolfyta) [reply 3-blank/README.md:95]: Ye but it don't make sense... How its currently written leaves `Discord Server` undefined

Comment (VictoriqueMoe) [reply 3-blank/README.md:95]: @TeamWolfyta is right. "Ask in Discord server" doesn't make sense

Comment (oceanroleplay) [reply 3-blank/README.md:95]: the abstract is, ``Ask`` in ``discord server``, point out, ``our`` and `join` does not make much sense.

Comment (oceanroleplay) [reply 3-blank/README.md:95]: Maybe, ``Ask in Community Discord Server`` would be more accurate.

Comment (TeamWolfyta) [reply 3-blank/README.md:95]: > Maybe, `Ask in Community Discord server` would be more accurate.

`Ask in the community Discord server`

Comment (oceanroleplay) [reply 3-blank/README.md:95]: ok

File: 1-starter/README.md [hunk 2]
```diff
@@ -92,7 +92,7 @@ For the full command list please view the [Docker Documentation](https://docs.do
 
 - [Check frequently asked questions](https://discord-ts.js.org/docs/faq)
 - [Check examples](https://github.com/oceanroleplay/discord.ts/tree/main/packages/discordx/examples)
-- Join and Ask in our [Discord Server](https://discord-ts.js.org/discord)
+- Ask in the community [Discord Server](https://discord-ts.js.org/discord)
```
Comment (oceanroleplay) [inline 1-starter/README.md:95]: change all ``Discord Server`` to ``Discord server``

Comment (TeamWolfyta) [reply 1-starter/README.md:95]: Missed a few, oops

File: 1-starter/README.md [hunk 3]
```diff
@@ -88,8 +92,8 @@ For the full command list please view the [Docker Documentation](https://docs.do
 
 - [Check frequently asked questions](https://discord-ts.js.org/docs/faq)
 - [Check examples](https://github.com/oceanroleplay/discord.ts/tree/main/packages/discordx/examples)
-- Ask in [Discord Server](https://discord-ts.js.org/discord)
+- Ask in the community [Discord server](https://discord-ts.js.org/discord)
 
 # 💖 Thank you
 
-Show your support for [Discordx](https://www.npmjs.com/package/discordx) by giving us a star on [GitHub](https://github.com/oceanroleplay/discord.ts).
+Why not show your support for [Discordx](https://www.npmjs.com/package/discordx) by giving us a star on [GitHub](https://github.com/oceanroleplay/discord.ts).
```
Comment (oceanroleplay) [inline 1-starter/README.md:99]: revert

Comment (TeamWolfyta) [reply 1-starter/README.md:99]: Old message sounds more like a command than something the reader could do.

Comment (oceanroleplay) [reply 1-starter/README.md:99]: `You can support Discordx by giving it a GitHub star`?

Comment (TeamWolfyta) [reply 1-starter/README.md:99]: Sounds better. I'll use that instead, it keeps the tone of the original while not sounding like a command xd

Comment (oceanroleplay) [reply 1-starter/README.md:99]: same would be needed on `discord.ts` repo as well

**Review by oceanroleplay — APPROVED** [review]

File: 1-starter/README.md (hunk 3 above)
Comment (oceanroleplay) [reply 1-starter/README.md]: for `discord.ts` you will fine a simillar line in ``create-discordx`` source code, we should cover it as well

**Pull request merged**